test/test_input_recipeset.py

```diff
@@ -1390,3 +1390,171 @@ def testNewBehaviour(self):
 
         pkg = self.generate().walkPackagePath("root")
         self.assertTrue(pkg.getCheckoutStep().getScmList()[0].getProperties(False)["prune"])
+
+
+class TestScmDefaults(RecipesTmp, TestCase):
+    """ Test behaviour of pruneImportScm policy"""
+
+    def testScmDefaultsGit(self):
+        """Test scmDefaults for gitscm"""
+        with TemporaryDirectory() as tmp:
+            os.chdir(tmp)
+            os.mkdir("recipes")
+
+            self.writeRecipe("root", """\
+                root: True
+                checkoutSCM:
+                   scm: git
+                   url: foo@bar
+                buildScript: "true"
+                packageScript: "true"
+                """)
+            pkg = self.generate().walkPackagePath("root")
+            self.assertEqual(pkg.getCheckoutStep().getScmList()[0].getProperties(False)["branch"],
+                    "master")
+            self.assertEqual(pkg.getCheckoutStep().getScmList()[0].getProperties(False)["sslVerify"],
+                    None)
```

Comment:
How about this to reduce the redundancy?

```suggestion
            props = pkg.getCheckoutStep().getScmList()[0].getProperties(False)
            self.assertEqual(props["branch"], "master")
            self.assertEqual(props["sslVerify"], None)
```
...